cli/src/main/java/org/apache/falcon/cli/FalconInstanceCLI.java

```diff
@@ -230,7 +247,7 @@ public void instanceCommand(CommandLine commandLine, FalconClient client) throws
             validateNotEmpty(colo, FalconCLIConstants.COLO_OPT);
             validateNotEmpty(start, FalconCLIConstants.START_OPT);
             validateNotEmpty(type, FalconCLIConstants.TYPE_OPT);
```

Comment:
For the sake of consistency, should we move the validateNotEmpty to ValidationUtil as well?